### PR TITLE
Rooms

### PR DESCRIPTION
**Pull request by SergeyShynkarenko**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -53,16 +53,61 @@ <h2 class="header__title-hotels"><span>LUXURY</span> HOTELS</h2>
...
+						<img src="images/dest/arrow-scroll.svg" alt="Arrow">
+					</a>
+				</div>
 			</div>
-			<div class="scroll__btn">
-				<p>Scroll</p>
-				<a href="#">
-					<img src="images/dest/arrow-scroll.svg" alt="Arrow">
-				</a>
+
```
Comment (IShinkarenko) [inline app/index.html:62]: remove space

File: app/scss/pages/_home.scss [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+		text-align: center;
+		@media (max-width : 680px) {
+			margin-bottom: 25px;
+		}
+		@media (max-width : 480px) {
+			font-size: 17px;
+		}
+	}
+}
+.room {
```
Comment (IShinkarenko) [inline app/scss/pages/_home.scss:28]: always add space when start new styles

File: app/scss/pages/_home.scss [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+			height: 450px;
+			@media (max-width : 990px) {
+				max-width: 100%;
+				height: auto;
+			}
+		}
+	}
+}
+
+.room__views {
```
Comment (IShinkarenko) [inline app/scss/pages/_home.scss]: why did you start new class instead **&__view?**

File: app/scss/pages/_home.scss [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+			text-align: center;
+			@media (max-width : 990px) {
+				margin: 20px auto;
+			}
+		}
+	}
+	&__img {
+		img {
+			width: 650px;
+			height: 450px;
```
Comment (IShinkarenko) [inline app/scss/pages/_home.scss]: incorrect
You have a case when you have two different images with differtnt width and height but you should make them equal witdh and height
Look at this article - https://medium.com/@kimcodeashian/simple-css-solutions-how-to-resize-images-with-different-dimensions-without-cropping-eb744e93246d

Screenshot 2021-01-24 at 22 30 40

File: app/scss/pages/_home.scss [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+	}
+	@media (max-width : 480px) {
+		margin-bottom: 35px;
+	}
+	&__info {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		margin-right: 70px;
+		@media (max-width : 990px) {
```
Comment (IShinkarenko) [inline app/scss/pages/_home.scss:45]: Screenshot 2021-01-24 at 22 34 56

Not really good
Need to make images a bit less and text will be wide everything should look good

File: app/scss/pages/_home.scss [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+		}
+	}
+	@media (max-width : 990px) {
+		flex-wrap: wrap;
+		justify-content: center;
+	}
+	@media (max-width : 480px) {
+		margin-bottom: 35px;
+	}
+	&__info {
```
Comment (IShinkarenko) [inline app/scss/pages/_home.scss:39]: Screenshot 2021-01-24 at 22 37 51

see the picture

Better when on this resolution picture will be first then description and then button

**Review by IShinkarenko — APPROVED** [review]

File: app/scss/pages/_home.scss (hunk 2 above)
Comment (SergeyShynkarenko) [reply app/scss/pages/_home.scss:28]: done

File: app/scss/pages/_home.scss (hunk 3 above)
Comment (SergeyShynkarenko) [reply app/scss/pages/_home.scss]: done

File: app/scss/pages/_home.scss (hunk 6 above)
Comment (SergeyShynkarenko) [reply app/scss/pages/_home.scss:39]: done

File: app/index.html (hunk 1 above)
Comment (SergeyShynkarenko) [reply app/index.html:62]: done

File: app/scss/pages/_home.scss (hunk 5 above)
Comment (SergeyShynkarenko) [reply app/scss/pages/_home.scss:45]: done

File: app/index.html (hunk 1 above)
Comment (SergeyShynkarenko) [reply app/index.html:62]: done

Comment (SergeyShynkarenko) [reply app/index.html:62]: done

**Review by IShinkarenko — APPROVED** [review]

**Pull request merged**